templates/_layouts/docs.html

```diff
@@ -155,7 +155,13 @@
       </aside>
 
       <main class="col-9" id="main-content">
-        {% block content %}{{ content | safe }}{% endblock content %}
+        {% block content %}
+          <h1>{{ title.split('|')[0] | safe }}</h1>
```

Comment:
I think we can drop `I safe` here, we don't want any HTML code passed from title to the page (there shouldn't be any HTML there anyway), so this should be enough:

```suggestion
          <h1>{{ title.split('|')[0] }}</h1>
```

Reply:
Pushed a fix for the double titles problem and removed `safe`.